devito/symbolics/extended_sympy.py

```diff
@@ -410,14 +422,14 @@ class Literal(sympy.Symbol):
 MIN = Function('MIN')
 
 cast_mapper = {
-    np.int: INT,
+    int: INT,
     np.int32: INT,
     np.int64: INT,
     np.float32: FLOAT,
     float: DOUBLE,
     np.float64: DOUBLE,
 
-    (np.int, '*'): INTP,
+    (int, '*'): INTP,
```

Comment:
?

Reply:
`np.int` is deprecated in numpy, you should use `int` instead (as per warning message). Or, much better, `np.int32` or `np.int64`, as one usually does

Reply:
IS it deprecated in the latest version or all versions we support? (>1.16)

Reply:
don't know honestly. I know that some CI logs contain a long list of warnings, some of which due to this thing